Source/WebCore/dom/Microtasks.h

```diff
@@ -34,6 +34,31 @@ class VM;
 
 namespace WebCore {
 
+class MicrotaskCheckpointScope final {
+    WTF_FORBID_HEAP_ALLOCATION;
+    WTF_MAKE_NONCOPYABLE(MicrotaskCheckpointScope);
+    WTF_MAKE_NONMOVABLE(MicrotaskCheckpointScope);
+public:
+    explicit MicrotaskCheckpointScope(EventLoop& eventLoop)
+    {
+        Ref protectedEventLoop = eventLoop;
```

Comment:
This is unnecessary. Because `eventLoop` is a function argument, the caller must keep it alive by our safer C++ guideline.